projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -236,17 +230,6 @@ function JetpackContactFormEdit( { name, attributes, setAttributes, clientId, cl
 							instanceId={ instanceId }
 						/>
 					) }
-					{ ! isFormModalEnabled && ! isSimpleSite() && canUserInstallPlugins && (
-						<>
-							<AkismetPanel />
-							<PanelBody title={ __( 'CRM connection', 'jetpack-forms' ) } initialOpen={ false }>
-								<CRMIntegrationSettings jetpackCRM={ jetpackCRM } setAttributes={ setAttributes } />
-							</PanelBody>
-							<PanelBody title={ __( 'Creative Mail', 'jetpack-forms' ) } initialOpen={ false }>
-								<NewsletterIntegrationSettings />
-							</PanelBody>
-						</>
-					) }
```

Comment:
Removing this block is the starting point. Every file that's removed is either these components, or other files/components used only by these components. 

Note that we had checks for simple sites and ! isSimpleSite() && canUserInstallPlugins on this. I've moved those checks to the IntegrationControls just above as I think they should still be there.